test_libs/pyspec/eth2spec/test/phase_0/block_processing/test_process_attestation.py

```diff
@@ -274,11 +275,14 @@ def test_bad_crosslink_end_epoch(spec, state):
 
 @with_all_phases
 @spec_state_test
-def test_inconsistent_bitfields(spec, state):
+def test_inconsistent_bits(spec, state):
     attestation = get_valid_attestation(spec, state)
     state.slot += spec.MIN_ATTESTATION_INCLUSION_DELAY
 
-    attestation.custody_bitfield = deepcopy(attestation.aggregation_bitfield) + b'\x00'
+    custody_bits = deepcopy(attestation.aggregation_bits)
```

Comment:
pythonic method to copy list
```suggestion
    custody_bits = attestation.aggregation_bits[:]
```